src/snipe/cli/cli_qc.py

```
@@ -14,6 +14,8 @@
 from snipe.api.snipe_sig import SnipeSig
 from snipe.api.reference_QC import ReferenceQC
 
+import concurrent.futures
```

Ruff

Redefinition of unused `concurrent` from line 10

Remove definition: `concurrent`

(F811)